esbuilder/lit-connect-modal/esbuild.js

```diff
@@ -0,0 +1,26 @@
+const { build } = require('esbuild');
+const path = require('path');
+const fs = require('fs');
+
+const entryPoint = path.resolve(__dirname, 'src/modal.js');
+const outputDir = 'packages/auth-browser/src/lib/connect-modal/modal.ts';
+
+build({
+  entryPoints: [entryPoint],
+  bundle: true,
+  // minify: true,
+  sourcemap: false,
```

Comment:
can this be a configurable  somehow? if we dont bundle sourcemaps for this module then it wont be debuggable

Reply:
the output is in `./packages/authBrowser/src/` you don't wanna create a source map there to be bundled